azurerm/internal/services/appconfiguration/app_configuration_resource.go

```diff
@@ -239,6 +269,11 @@ func resourceArmAppConfigurationUpdate(d *schema.ResourceData, meta interface{})
 		Tags: tags.Expand(d.Get("tags").(map[string]interface{})),
 	}
 
+	if _, ok := d.GetOk("identity"); ok {
+		appConfigIdentity := expandAppConfigurationIdentity(d)
+		parameters.Identity = appConfigIdentity
```

Comment:
Some simplification:
```suggestion
		parameters.Identity = expandAppConfigurationIdentity(d)
```